src/__tests__/fromVariablesBruker.test.ts

```diff
@@ -4,7 +4,8 @@ import { MeasurementXYVariables } from 'cheminfo-types';
 import { convert } from 'jcampconverter';
 import { xMultiply } from 'ml-spectra-processing';
 
-import { fromVariables, JcampOptions } from '..';
+import { JcampOptions } from '..';
+import from1DNMRVariables from '../from1DNMRVariables';
```

Comment:
Because in this file we only test from1DNMRVariables, please rename the file to from1DNMRVariables.test.ts